onnxruntime/test/providers/tensorrt/tensorrt_basic_test.cc

```diff
@@ -277,7 +277,7 @@ TEST(TensorrtExecutionProviderTest, SessionCreationWithSingleThreadAndInferenceW
 }
 
 // Test loading same model in different way, when hash id is generated via model name/model content/env metadata
-TEST(TensorrtExecutionProviderTest, TRTMetadefIdGeneratorUsingModelHashing) {
+TEST(TensorrtExecutionProviderTest, TRTModelIdGeneratorUsingModelHashing) {
```

Comment:
Yes, that's something we discussed internally. If user loads model bytes rather than onnx model, a new engine will be generated. I will update ORT document to reflect this.